### PR TITLE
Update subversion and git plugins for security patches

### PR DESCRIPTION
**Pull request by ramereth**
This also included updating/adding a few other dependant plugins to get this
working properly.

### CODE REVIEW

File: attributes/plugins.rb [hunk 1]
```diff
@@ -5,13 +5,15 @@
 #     println ("${plugin.getShortName()}:${plugin.getVersion()}")
 # }
 default['osl-jenkins']['restart_plugins'] = %w(
-  credentials:2.1.13
+  structs:1.14
+  credentials:2.1.16
```
Comment (doublej472) [inline attributes/plugins.rb]: It looks like version [2.1.15](https://plugins.jenkins.io/credentials) might end up breaking things, will this need to be handled?

Comment (ramereth) [reply attributes/plugins.rb]: I'm not sure I'm following what you're referring to here.

Comment (doublej472) [reply attributes/plugins.rb]: This is specifically what I'm referencing:
```
NOTE: all existing credentials fingerprint tracking history will be lost.
```

Comment (ramereth) [reply attributes/plugins.rb]: I don't think we need to worry about that.

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.9.19.  Have a nice day!

**Pull request merged**